docs/data/sdks/android/index.md

```diff
@@ -9,7 +9,7 @@ icon: material/android
 
 This is the official documentation for the Amplitude Analytics Android SDK.
 
-!!!attention "Legacy SDK"
+!!!deprecated "Legacy SDK"
     This SDK is legacy and only continue to receive bug fixes until deprecated. Checkout and upgrade to the [Android Kotlin SDK](../android-kotlin) which supports plugins, SDK integrations and more.
```

Comment:
```suggestion
    This is a legacy SDK and will only receive bug fixes until deprecation. Upgrade to the [Android Kotlin SDK](../android-kotlin) which supports plugins, SDK integrations, and more.
```

Reply:
Changed for android and other SDKs that has Legacy block